tips/025_tip_create_new_authority.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: How to create a new Authority
+sitemap:
+priority: 0.1
+lastmod: 2018-10-05T18:20:00-00:00
+---
+# How to create a new Authority
+
+Tip submitted by @Tonterias
+
+Let's say that you need a new authority besides the given ones of ADMIN and USER.
+
+Modify /src/main/java/com/jhipsterpress/web/security/AuthoritiesConstants.java file to include your new authorities:
```

Comment:
Don't put your package name here, as it's different for everyone. Just write `AuthoritiesConstants`, people will find it with their IDE

Reply:
Ok, I explain it for newbies lke me, so I tend to be super detailed. Should I change it in a new Pull Request or have you changed it? This is my second PR. Thanks for your help.